code/modules/mob/living/carbon/xenomorph/castes/king/king.dm

```diff
@@ -53,9 +53,11 @@
 	. = ..()
 	ownerhive = hivenumber
 	addtimer(CALLBACK(src, .proc/choose_king), KING_SUMMON_TIMER_DURATION)
+	SSmonitor.stats.king++
 
 /obj/structure/resin/king_pod/Destroy()
 	. = ..()
+	SSmonitor.stats.king--
```

Comment:
doenst this get destroyed when king spawns ?

Reply:
yes